content/blog/2021-07-16-july-21-dvc-heartbeat.md

```diff
@@ -47,8 +47,9 @@ a model, tracking changes to data and model, dvc
 gap in visualizations using <span style="color:purple">**S**</span>treamlit. You
 won't want to miss this one!
 
-![DVC and Streamlit](/uploads/images/2021-07-16/streamlit2.png) _DVC + Streamlit
-= ♥️! [Source link](https://www.sicara.ai/blog/dvc-streamlit-webui-ml)_
+![DVC and Streamlit](/uploads/images/2021-07-16/streamlit2.png '=700') _DVC +
```

Comment:
where do we get this image from initially?

Reply:
From here 👇 Line 52. The blog post.

Reply:
Got it, thanks. Just a quick note  - stretching images means they look a bit blurry. I hope it's fine in this case.